src/quo2/components/settings/settings_item/component_spec.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.settings.settings-item.component-spec
+  (:require [quo2.components.settings.settings-item.view :as settings-item]
+            [test-helpers.component :as h]))
+
+(def props
+  {:title               "Account"
+   :accessibility-label :settings-item
+   :action              :arrow
+   :image               :icon
+   :description         :none
+   :tag                 :none
+   :label               :none
+   :blur?               false})
+
+(h/describe "Settings list tests"
+  (h/test "Default render of Setting list component"
+    (h/render [settings-item/view props])
+    (h/is-truthy (h/get-by-label-text :settings-item)))
+
+  (h/test "It renders a title"
+    (h/render [settings-item/view props])
+    (h/is-truthy (h/get-by-text "Account")))
+
+  (h/test "its gets passed an on press event"
+    (let [event (h/mock-fn)]
+      (h/render [settings-item/view
+                 (merge props {:on-press event})])
```

Comment:
nit, let's use assoc

Reply:
using `assoc` for this particular one is fine, but `merge` is used multiple times in this file to merge maps, so for consistency I think we can keep it merge